src/tox/Network/Tox.lhs

```diff
@@ -26,10 +26,76 @@ lengths (e.g.  \texttt{4 | 16}), or an inclusive range (e.g. \texttt{[0,
 100]}).  Open ranges are denoted \texttt{[n,]} to mean a minimum length of
 \texttt{n} with no specified maximum length.
 
+\section{TODO: Goals and threat model}
+
+This section should give an idea on what are the goals and non-goals of Tox, so
+that reader
+
+\begin{itemize}
+  \item understands what problems Tox intends to solve
+  \item can validate if they are addresed by this specification
+  \item can make better tradeoffs and decisions in his own reimplementation of the
+    protocol
+\end{itemize}
+
+(TODO: this is just a placeholder; some more technical description should be
+given)
+
+What Tox Does:
+
+\begin{itemize}
+  \item Authentication: user's address is its public key, thus "adding friend"
+    actually means verifying key (false in case of toxme?); drawback is that
```

Comment:
Here I disagree. If a website uses super secure backends to store the data, but has doors wide open in the process of information transfer, then I would not call this service/website secure. I could call components of it secure. So it should still be mentioned prominently in various places, that using toxme is not providing the same level of security as using a tox id.